ejb/src/main/resources/schema/jboss-ejb-iiop_1_0.xsd

```diff
@@ -0,0 +1,152 @@
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<xs:schema xmlns="urn:iiop" xmlns:javaee="http://java.sun.com/xml/ns/javaee"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           attributeFormDefault="unqualified" elementFormDefault="qualified"
+           targetNamespace="urn:iiop" version="1.0"
+           xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://www.jboss.org/j2ee/schema/jboss-ejb3-spec-2_0.xsd">
+   <xs:import namespace="http://java.sun.com/xml/ns/javaee" schemaLocation="http://www.jboss.org/j2ee/schema/jboss-ejb3-spec-2_0.xsd"/>
+
+   <xs:element name="iiop" type="iiopConfigType" substitutionGroup="javaee:assembly-descriptor-entry"/>
+
+   <xs:complexType name="iiopConfigType">
+      <xs:complexContent>
+         <xs:extension base="javaee:jboss-assembly-descriptor-bean-entryType">
+            <xs:annotation>
+               <xs:documentation>
+                  <![CDATA[
+                  The iiop element contains the IIOP settings that are to be applied to an EJB. It must identify the
+                  bean to which the settings apply via the ejb-name sub-element. A wildcard "*" is allowed to convey
+                  that the settings are to be applied to all EJBs in the deployment.
+
+                  The binding-name sub-element can be used to specify the name that must be used to bind the EJB IOR
+                  in the COSNaming service. If a wildcard "*" is used as bean-name, this setting is ignored.
+
+                  The ior-security-config sub-element contains the security settings that are to be inserted into the
+                  EJB IOR. These include transport config requirements, authentication service settings, and secure
+                  attribute service settings. If a wildcard "*" is used as bean-name, the IOR settings will be applied
+                  to all beans in the deployment, except for the ones that override these settings in their own iiop
+                  element.
+
+                  Prior to this version, ejb-name was a member of the iiopConfigType which was in error. ejb-name is provided
+                  as part of the extension base, jboss-assembly-descriptor-bean-entrytype.
+                  ]]>
+               </xs:documentation>
+            </xs:annotation>
+            <xs:sequence>
+              <!--Should not be provided, BZ-1192951 -->
```

Comment:
Ok, I looked at that thing more than a dozen times in varying browsers and the only thing I could see is a two space formatting difference in one comment line and one element. That's it, two spaces, it has been fixed and pushed.